lc0_main.go

```diff
@@ -384,6 +381,16 @@ func train(httpClient *http.Client, ngr client.NextGameResponse,
 	c := createCmdWrapper()
 	c.Version = "v0.10"
```

Comment:
It makes sense to increment version to v0.11.
Although it would be more useful if server also supported upper limit of client, not only lower.

Reply:
Do we want to scrape a version number out of lc0?  Might be nice to have the client && lc0 have the same version string.

I'll change the server code to accept a range of versions in another cl.